### ISSUE TITLE
[BUG] - 413 Error on `Logbook`

### ISSUE THREAD

**Issue by jordanhavard** [invernyx/smartcars-3-bugs#153]
### Describe the bug

When viewing the logbook, if a pilot has a large number of PIREPs, all of them are being requested at once sometimes causing a 413 error. Potentially implement a paging solution 

### How do you reproduce this bug?

Very hard for me to consistently repo

- I just clicked on the logbook and saw it in the console and the refresh icon spun forever

### Expected behavior

No 413 error

### Screenshots

Didn't manage to grab one

### Operating system

Windows 10 pro 10.0.19044

### Community airline

QVirtual

### smartCARS Version

0.8.1.2

### Plugins installed

all except simbrief

**Comment by GenericNerd** [comment]
Our API does not provide a 413 by itself. It is possible that this error is specific to your hosting/API solution. Can you provide the log for this?

**Comment by jordanhavard** [comment]
Would it not be within the axios library ? 

No error logs on my side matching. 
Very hard to repo, thought I'd raise and see fi you have anything on your side. Happy to monitor and report back

**Comment by GenericNerd** [comment]
How many PIREPs do you have?

**Comment by jordanhavard** [comment]
Our top pilot has over 3000 (and this is what I was testing on)

**Comment by turbofandude** [comment]
We have implemented a system similar to the flight center as of 0.9. The system can only report ~100 PIREPS and we have added a date search field as well. This way, a user can see either their most recent 100 PIREPS (by default) or search for specific PIREPS based on certain criteria to narrow it down.

All that is to say it will no longer be possible to have the client pull 1000s of PIREPS anymore.
